versions.json

```diff
@@ -250,6 +250,22 @@
         ],
         "fpm_directories": ["bin","include","lib"]
     },
+    "mungefs": {
+        "commitish": "2d7a4f6ff56e64b560f582a21438162a1e8f36ab",
```

Comment:
we should have a release tag of mungefs to use from this particular commit (currently tip of master) - need to release that as v1.0.2 and reference that here instead...